media/Config/reset.css

```diff
@@ -0,0 +1,30 @@
+html {
+    box-sizing: border-box;
+    font-size: 13px;
+  }
```

Comment:
this is closed at column 2 and all the below lines start at column 2..
seems strange

Reply:
Could I ask what is column meaning? It doesn't have specific format. I think it's just my mistakes.

Reply:
row-column... `}` is at 2'nd column (um... 0 base)

Reply:
I understand now. I'll fix it. thank you.